packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/README.md

```diff
@@ -2,19 +2,3 @@
 
 ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
 Please use it with caution.
-
```

Comment:
Maybe this can stay here and you can just update the twig structure example.

```twig
<EmptyState>
  <EmptyStateSection>
  ...
```

Reply:
https://github.com/lmc-eu/spirit-design-system/pull/1481#discussion_r1647432497